mobile/library/kotlin/io/envoyproxy/envoymobile/EngineBuilder.kt

```diff
@@ -559,6 +561,18 @@ open class EngineBuilder(
     return this
   }
 
+  /**
+   * Add virtual cluster configuration.
+   *
+   * @param cluster the structured configuration for a virtual cluster.
+   *
+   * @return this builder.
+   */
+  fun addVirutlaClusterConfig(config: VirtualClusterConfig): EngineBuilder {
```

Comment:
Did not review this code but noticed a typo in here `Virutla`

Reply:
Also, I wonder whether this method should not take a list of configs instead of a single config to help with cases where we have a bunch of them and do not want to call this method multiple times.

After thinking about this more... Since other methods such as `addPlatformFilter` take a single argument (not a list) I think that it's fine to not take a list in here too.